wagtail_advanced_form_builder/blocks/fields/base_field_block.py

```diff
@@ -39,14 +36,21 @@ class BaseFieldBlock(blocks.StructBlock):
     )
 
     max_length = blocks.IntegerBlock(
-        required=False,
-        help_text='Set a maximum length for this field. e.g. 100'
+        required=False, help_text="Set a maximum length for this field. e.g. 100"
     )
 
     default_value = blocks.CharBlock(
+        max_length=255, required=False, help_text="Set a default value for this field."
+    )
+    salesforce_object_name = blocks.CharBlock(
+        max_length=255,
+        required=False,
+        help_text="The Salesforce object name that the form field value will be integrated to.",
+    )
+    salesforce_api_field_name = blocks.CharBlock(
         max_length=255,
         required=False,
-        help_text='Set a default value for this field.'
+        help_text="The Salesforce API field name that the form field value will be integrated to.",
```

Comment:
a little black formatting in this file because it wants double quotes, but here's the real change